tietokanta/src/main/resources/db/migration/R__Laskutusyhteenveto_mhu.sql

```diff
@@ -1152,12 +1152,14 @@ BEGIN
                 l.erapaiva
             FROM kulu l
             JOIN kulu_kohdistus lk ON lk.kulu = l.id
-            JOIN toimenpideinstanssi tpi ON lk.toimenpideinstanssi = tpi.id 
+            JOIN toimenpideinstanssi tpi ON lk.toimenpideinstanssi = tpi.id
+            LEFT JOIN tehtavaryhma tr ON lk.tehtavaryhma = tr.id
             WHERE lk.tyyppi = 'muukulu'
                 AND lk.poistettu IS NOT TRUE
                 AND l.erapaiva BETWEEN hk_alkupvm AND aikavali_loppupvm
                 AND l.urakka = ur
                 AND tpi.id = t.tpi
+                AND tr.yksiloiva_tunniste != 'a6614475-1950-4a61-82c6-fda0fd19bb54' -- J - Johto- ja hallintokorvaus myös muuksi kuluksi kirjattuna huomioidaan laskutusyhteenvedon Hoidon johto-osion Johto- ja hallintokorvaus-rivillä.
```

Comment:
Tämmöset vertailut saattaa olla vähä kinkkisiä.
Mä veikkaan, että tuo haku jättää palauttamatta kaikki, joissa tr.yksiloiva_tunniste IS NULL.
Eli tämä olisi varmaankin parempi muuttaa muotoon:
AND (tr.yksiloiva_tunnista IS NOT NULL AND tr.yksiloiva_tunniste != 'a6614475-1950-4a61-82c6-fda0fd19bb54' OR tr.yksiloiva_tunniste IS NULL)